first if-statement in steamauth/userInfo.php

AO-IO opened the pull request:
first if-statement in steamauth/userInfo.php should be edited to check $_SESSION['steamid']

SmItH197 committed:
Merge pull request #55 from SmItH197/revert-49-patch-1

Revert "MySQL-Injection fix"

SmItH197 committed:
Added note about SQL

MeLlamoPablo committed:
A few improvements

-Muted file_get_contents() so that the API key is not publicly exposed
-userInfo.php won't do anything if the user hasn't logged in
-Added the constant STEAMAUTH_LOGOUT_LINK to make logout more flexible.

MeLlamoPablo committed:
Improved error message

MeLlamoPablo committed:
Removed constant, removed echo()

SmItH197 committed:
Merge pull request #65 from MeLlamoPablo/master

A few improvements

SmItH197 committed:
Updated version number

MeLlamoPablo committed:
Added support for passing _GET variables upon login

MeLlamoPablo committed:
Added support for http_build_query()

MeLlamoPablo committed:
Updated README.md with new information

MeLlamoPablo committed:
Fixed typo

SmItH197 committed:
Merge pull request #66 from MeLlamoPablo/master

Added support for passing _GET variables upon login

SmItH197 committed:
Updated version number

Commit:
URL-Generation fix, remove die()

The die() in userInfo.php made it possible for the complete page to stop
working without the option for displaying a error message. Librarys
should never die() in cases where it is out of the control of the
developer.

Commit:
D:

SmItH197 committed:
Merge pull request #72 from BlackCetha/master

URL-Generation fix, remove die()

SmItH197 committed:
Updated Readme

Commit:
Fixing demo page login button

the login button is not displaying due to a missing "echo"

SmItH197 committed:
Merge pull request #78 from beuted/patch-1

Fixing demo page login button

SmItH197 committed:
Fixed $steamauth['loginpage'] not working #80

SmItH197 committed:
loginpage check always returns true

Accidentally used isset(), which would always return true. So changed it to check if its empty or not. See #80 to see why I implemented this check.

SmItH197 committed:
Updated version number

SmItH197 committed:
Reverted to f47fc78056081d6a83d277ae447c5386dc0909fc

SmItH197 committed:
Fixed Demo Link

Obilux committed:
Remove second session_start();

We have used already.(At line 3.)

SmItH197 committed:
Merge pull request #88 from Obilux/patch-2

Remove second session_start();

andrewda committed:
Fix minor spelling errors

SmItH197 committed:
Merge pull request #102 from andrewda/patch-1

Fix minor spelling errors in demo.php

bman46 committed:
Update demo.php

added version # for easier troubleshooting

bman46 committed:
Update example.php

Again... Version number

bman46 committed:
Update settings.php

Added version

bman46 committed:
Update README.md

In a issue someone posted, i think that they thought to add the:
    <?php

    require 'steamauth/steamauth.php';
    
    ?>
at the top of the settings page, so i added this to clear it up so that they will understand it better.

bman46 committed:
Update README.md

Added a comma to original fix. also found error number: #106

SmItH197 committed:
Merge pull request #108 from bman46/patch-2

Version Numbers

SmItH197 committed:
Merge pull request #109 from bman46/patch-1

Version Numbers

SmItH197 committed:
Merge pull request #110 from bman46/patch-3

Clearing up instructions

RedSparr0w committed:
Update steamlogin() Function

Update steamlogin() Function to reload refering page if no "loginpage" specified

RedSparr0w committed:
Readability

made demo.php file a bit tidier and easier to read
also removed jQuery and Bootstrap scripts as they are not being used anywhere on the page

RedSparr0w committed:
Update Functions | better logout

Removed logout.php file - changed to function in steamauth.php

Added
- loginbutton() | Will display login button
- if (isset($_GET['login'])) | will log user in (regardless of page)
- if (isset($_GET['logout'])) | will log user out (regardless of page)
- if (isset($_GET['update'])) | will update user info (regardless of
page)

if no login/logout page set will refresh current page by default

RedSparr0w committed:
Move version info to end of file

RedSparr0w committed:
Destroy Session on logout

Destroy / Unset Session vars on logout

RedSparr0w committed:
Update Readme to include php style code blocks

SmItH197 committed:
Merge pull request #117 from RedSparr0w/demo

Readability of demo

SmItH197 committed:
Revert "Readability of demo"

SmItH197 committed:
Merge pull request #124 from SmItH197/revert-117-demo

Revert "Readability of demo"

SmItH197 committed:
Fixed merge conflict

SmItH197 committed:
Merge branch 'RedSparr0w-master'

SmItH197 committed:
Merge #117

SmItH197 committed:
Renamed to avoid file conflicts #120

SmItH197 committed:
Alternative if headers already sent

SmItH197 committed:
Update settings filename

SmItH197 committed:
Fix formatting

SmItH197 committed:
Fixed demo.php

SmItH197 committed:
Updated version number

SmItH197 committed:
Changed demo site link

RedSparr0w committed:
Merge pull request #1 from SmItH197/master

merge v3.1.1

RedSparr0w committed:
updating information | include to require

- Added information on how to update user info

- Changed `$_SESSION['steam_uptodate']` to be timestamp rather than true
- can be used to update user info if older than 1 day for example
- Removed 1 level of indentation
- changed include to require as it is required in order to function
correctly
- moved require inside `if` as not needed otherwise

- changed includes to requires as they are required to function
correctly
- [removed brackets from
require](http://php.net/manual/en/function.require.php#75875)
- some indentation changes
- update user info if older than 24 hours
- added `exit;` to update and logout

- add `$steamprofile['steam_uptodate'] ` to demo.php page
- close `<td>` on same line

RedSparr0w committed:
move require openid.php

only require openid.php if $_GET['login'] set, otherwise not needed for
anything

RedSparr0w committed:
create $steamauth['updateinterval']

create `$steamauth['updateinterval']` and require SteamConfig.php at
start of steamauth.php

RedSparr0w committed:
removed auto update user, added info to readme

bman46 committed:
Added check out the wiki

bman46 committed:
Added suggestion to check out demo.php

bman46 committed:
Added URLS

SmItH197 committed:
Merge #127 with edited readme

SmItH197 committed:
Merge branch 'RedSparr0w-master'

SmItH197 committed:
Merge #130 with fixed conflicts

SmItH197 committed:
Merge branch 'bman46-master'

RedSparr0w committed:
update readme - Issues information

Segeco committed:
Newbie Setup

Added some more info so newbz can set this thing up.

SmItH197 committed:
Gambling Update

SmItH197 committed:
Gambling Update typo

SmItH197 committed:
Licence.txt

SmItH197 committed:
New demo page!

SmItH197 committed:
Updated with new login buttons #148

SmItH197 committed:
Merge pull request #133 from RedSparr0w/patch-1

update readme - Issues information

SmItH197 committed:
Updated to Lightopenid v1.3.1

SmItH197 committed:
Updated version number

SmItH197 committed:
Updated version number

SmItH197 committed:
Updated version number

SmItH197 committed:
Updated version number

SmItH197 committed:
Updated version number

SmItH197 committed:
Fixed typo, formatted text

RedSparr0w committed:
support for https login button image

fixes #154

Segeco committed:
Update SteamConfig.php

Fixed string escape and 500 error.

SmItH197 committed:
Merge pull request #141 from Segeco/patch-1

Newbie Setup

SmItH197 committed:
Merge pull request #160 from RedSparr0w/patch-2

support for https login button image

creyD committed:
Minor markup correction

Just a minor readme markup correction.

Commit:
Create composer.json

Commit:
Added Description

SmItH197 committed:
Merge pull request #174 from creyD/master

Minor markup correction

SmItH197 committed:
Merge pull request #175 from MartinBecker/patch-1

Composer.json

Commit:
composer.json changed Package name to lowercase

Packagist only accepts lowercase letters in the composer.json

SmItH197 committed:
Merge pull request #184 from MartinBecker/patch-2

composer.json changed Package name to lowercase

SmItH197 committed:
Update demo link to https

BlackCetha commented:
I think you might've selected the wrong base commit/branch. You need to fork the master branch and edit only the file you want to change.

BlackCetha reviewed (changes requested):
Please select the current HEAD as base commit.

maxorlovsky committed:
fix(composer.json): remove version, add keywords, type, homepage parameters

bman46 committed:
Create ISSUE_TEMPLATE.md

I think that it would be a good addition and is recommended by the community standards. Would also help guide questions/issues so we don't have issues that are unclear.

SmItH197 committed:
Merge pull request #206 from maxorlovsky/master

fix(composer.json): remove version, add keywords, type, homepage parameters

SmItH197 committed:
Merge pull request #211 from bman46/patch-1

Create ISSUE_TEMPLATE.md

Commit:
use https

SmItH197 committed:
Merge pull request #218 from guuurgle/master

use https

SmItH197 committed:
Force https for login button

Force fetching the image for the login button over https, this should solve "Your site is not fully secure" style error (as your fetching resources over http).

bman46 committed:
Update SteamConfig.php

bman46 committed:
Update steamauth.php

bman46 committed:
Update userInfo.php

bman46 committed:
Update demo.php

bman46 committed:
Update example.php

SmItH197 committed:
Merge pull request #220 from bman46/master

Update Version Numbers

Commit:
Update demo.php

Closed comment properly.

SmItH197 committed:
Merge pull request #225 from DavidG103/master

Update demo.php

BlackCetha commented:
@SmItH197 Could you close this PR? It's been inactive and has the wrong base.
